ReportPortal.Client.Tests/LaunchItem/LaunchItemFixture.cs

```diff
@@ -265,47 +272,23 @@ public async Task TrimLaunchName()
         [Fact]
         public async Task StartForceFinishIncompleteLaunch()
```

Comment:
Test is broken, here we need verify that we can force finish a launch successfully.